services/filemanager/filemanager.go

```diff
@@ -44,7 +44,7 @@ type FileManager interface {
 	GetObjectNameFromLocation(string) (string, error)
 	GetDownloadKeyFromFileLocation(location string) string
 	DeleteObjects(ctx context.Context, keys []string) error
-	ListFilesWithPrefix(ctx context.Context, prefix string, maxItems int64) (fileObjects []*FileObject, err error)
+	ListFilesWithPrefix(ctx context.Context, startAfter, prefix string, maxItems int64) (fileObjects []*FileObject, err error)
```

Comment:
Given that `startAfter` is not always required, it is better to place it as the last argument.

```suggestion
	ListFilesWithPrefix(ctx context.Context, prefix string, maxItems int64startAfter, startAfter string) (fileObjects []*FileObject, err error)
```

Reply:
It is not required now, but the next PR (Replay supporting all object storage) will start using it